src/lib.js

```diff
@@ -604,6 +605,16 @@ function load_file(filename, options, n_tries)
             }
             else if(http.response)
             {
+                if(options.range)
+                {
+                    let enc = http.getResponseHeader("Content-Encoding");
+                    if(enc && enc !== "identity")
+                    {
+                        console.error("Range requested from compressed resource " + filename, enc);
+                        http.abort();
```

Comment:
This breaks if at one point in the future there is a new Content-Encoding that is compatible with range requests. I'd suggest removing the `http.abort()`, and just leave the warning.

It would be nice to improve the warning message, e.g.:

`console.warn("Server sent Content-Encoding in response to ranged request", { filename, enc });`